common/app/staticpages/StaticPages.scala

```diff
@@ -23,14 +23,34 @@ object StaticPages {
       iosType = None,
       shouldGoogleIndex = false))
 
-    def simpleEmailSignupPage(id: String, webTitle: String): SimplePage = SimplePage(
-      MetaData.make(
-        id = id,
-        section = Option(SectionSummary(id="global", activeBrandings=None)),
-        webTitle = webTitle,
-        analyticsName = "global",
-        // a currently running AB test is using `contentType = "survey"` change this to "signup" after 2016-09-13
-        contentType = "survey",
-        iosType = None,
-        shouldGoogleIndex = false))
+  def contributorEmailPage: SimplePage = SimplePage(
```

Comment:
I'll look at relocating it when we go live with the feature. It's here because I followed the same pattern as other pages that I knew worked.